guide/14-deep-learning/text-classification-workflow.ipynb

```diff
@@ -0,0 +1,1222 @@
+{
```

Comment:
<p>In the first bullet it entity extraction to be replaced with text classification.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
fixed